scripts/exrap_dump.sh

```diff
@@ -986,7 +987,7 @@ elif [ $cyc -eq 23 ]; then   # (22.50 - 23.49 Z)
 ###unset SKIP_006063 # reflectivity 23Z
 fi
 
-$ushscript_dump/bufr_dump_obs.sh $dumptime ${def_time_window_6} 1 nexrad
+$USHbufr_dump/bufr_dump_obs.sh $dumptime ${def_time_window_6} 1 nexrad
```

Comment:
This change is not required. The original $uscript_dump is defined in JOBSPROC_RAP_DUMP and can be retained. Please revert back to original code:
$ushscript_dump/bufr_dump_obs.sh $dumptime ${def_time_window_1} 1 nexrad

Reply:
revised:
$ushscript_dump/bufr_dump_obs.sh $dumptime ${def_time_window_6} 1 nexrad